diff --git a/dist/esm/lib/typescript/writeConfigurationDefaults.js b/dist/esm/lib/typescript/writeConfigurationDefaults.js
index 6af2b1116bce0bed3883939deb0fcc142c68e971..dbc7adf6404af4939eb23b48b766cdaf339beeae 100644
--- a/dist/esm/lib/typescript/writeConfigurationDefaults.js
+++ b/dist/esm/lib/typescript/writeConfigurationDefaults.js
@@ -115,6 +115,7 @@ export function getRequiredConfiguration(ts) {
     return res;
 }
 export async function writeConfigurationDefaults(ts, tsConfigPath, isFirstTimeSetup, hasAppDir, distDir, hasPagesDir) {
+    return;
     var _userTsConfig_compilerOptions;
     if (isFirstTimeSetup) {
         await fs.writeFile(tsConfigPath, '{}' + os.EOL);
diff --git a/dist/lib/typescript/writeConfigurationDefaults.js b/dist/lib/typescript/writeConfigurationDefaults.js
index 8f5b753b8f01b5f5961cdf2b8c725620c2869fc7..d0ea14d2b52528229e91da2133a6caee23176c52 100644
--- a/dist/lib/typescript/writeConfigurationDefaults.js
+++ b/dist/lib/typescript/writeConfigurationDefaults.js
@@ -183,6 +183,7 @@ function getRequiredConfiguration(ts) {
     return res;
 }
 async function writeConfigurationDefaults(ts, tsConfigPath, isFirstTimeSetup, hasAppDir, distDir, hasPagesDir) {
+    return;
     var _userTsConfig_compilerOptions;
     if (isFirstTimeSetup) {
         await _fs.promises.writeFile(tsConfigPath, '{}' + _os.default.EOL);
